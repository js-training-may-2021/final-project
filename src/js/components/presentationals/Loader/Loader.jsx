import React from 'react';

const Loader = () => {
	return (
		<p className="page__text page__text--message">Loading...</p>
	);
};

export default React.memo(Loader);
