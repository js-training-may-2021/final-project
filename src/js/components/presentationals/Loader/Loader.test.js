import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './Loader.jsx';

it(`Render Loader`, () => {
  const tree = renderer
    .create(
      <Loader />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
