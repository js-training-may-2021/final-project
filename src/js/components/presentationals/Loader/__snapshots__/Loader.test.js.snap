// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Loader 1`] = `
<p
  className="page__text page__text--message"
>
  Loading...
</p>
`;
